prototype/pca_compression/compress.py

```diff
@@ -0,0 +1,75 @@
+import serial
+import numpy as np
+from PIL import Image
+import re
+import matplotlib.pyplot as plt
+
+def encode_arduino_input(matrix):
```

Comment:
Do we want comments on our prototypes as well?

Reply:
What do you mean comments ? Like if we review the prototypes too or if we comment what functions do in prototypes